frontend/public/module/k8s/k8s.ts

```diff
@@ -47,7 +47,7 @@ export const referenceForOwnerRef = (ownerRef: OwnerReference): GroupVersionKind
   )(ownerRef.kind);
 
 export const referenceForModel = (model: K8sKind): GroupVersionKind =>
-  referenceForGroupVersionKind(model.apiGroup || 'core')(model.apiVersion)(model.kind);
+  referenceForGroupVersionKind(model?.apiGroup || 'core')(model?.apiVersion)(model?.kind);
```

Comment:
Discussed with @Divyansh42.
Let's revert this change and address the issue in `ResourceDropdown`

Reply:
Done.